src/DynamicRupture/Factory.cpp

```diff
@@ -50,12 +50,12 @@ Products NoFaultFactory::produce() {
 }
 
 Products LinearSlipWeakeningFactory::produce() {
-  return {
-      std::make_unique<seissol::initializers::LTS_LinearSlipWeakening>(),
-      std::make_unique<initializers::LinearSlipWeakeningInitializer>(drParameters),
-      std::make_unique<friction_law_impl::LinearSlipWeakeningLaw<friction_law::NoSpecialization>>(
-          drParameters),
-      std::make_unique<output::OutputManager>(new output::LinearSlipWeakening)};
+  return {std::make_unique<seissol::initializers::LTS_LinearSlipWeakening>(),
+          std::make_unique<initializers::LinearSlipWeakeningInitializer>(drParameters),
+          std::make_unique<
+              friction_law_impl::LinearSlipWeakeningLaw<friction_law_impl::NoSpecialization>>(
```

Comment:
dr/cpp has:
```std::make_unique<friction_law_impl::LinearSlipWeakeningLaw<friction_law::NoSpecialization>>(```

Reply:
yes.  `friction_law_impl::NoSpecialization` is done on purpose. `NoSpecialization` in `seissol::dr::friction_law::gpu` namespace may become a bit different in contrast to 
https://github.com/SeisSol/SeisSol/blob/de874bd540f4e1d064f75764547389537c0aa0f4/src/DynamicRupture/FrictionLaws/LinearSlipWeakening.h#L228-L242
once I add `BiMaterialFault` implementation for GPUs

Reply:
got it!